deploy/ocm-agent-operator-managednotifications/10-managednotifications-upgrade-cr.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotification
+metadata:
+  name: sre-upgrade-managed-notifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notifications:
+    - name: UpgradeControlPlaneUpgradeTimeout
+      severity: Error
+      summary: "Control plane upgrade completion delayed"
+      activeBody: |-
+        There has been a delay observed in the anticipated completion time of your cluster's Control Plane upgrade. Red Hat SRE has been notified of the situation.
```

Just need to remove the final trailing '.' as it will be inserted automatically by OCM SL.

```suggestion
        There has been a delay observed in the anticipated completion time of your cluster's Control Plane upgrade. Red Hat SRE has been notified of the situation
```